js/base-journal.js

```
@@ -122,6 +133,86 @@ function d20plusJournal () {
 			}
 		});
 
+		// "Set avatar" option
+		first.after(`<li class="b20-change-avatar" data-action-type="changeavatar">Set Avatar</li>`);
+		$("#journalitemmenu ul").on(window.mousedowntype, "li[data-action-type=changeavatar]", function () {
+			const id = d20plus.journal.lastClickedJournalItemId;
+			const item = d20.Campaign.characters.get(id) || d20.Campaign.handouts.get(id);
+			const name = item?.attributes.name || "Unnamed";
+			if (!item?.attributes.hasOwnProperty("name") || !item?.attributes.hasOwnProperty("avatar")) return;
```

🤔 would be nice to show an error message rather than silently doing nothing; Roll20 doesn't have e.g. a "toast" system, does it? If not, maybe we should bash a utility function together out of dialogue windows sometime

even an `alert` or a `console. ...` message would be better than nothing, for now